src/libre/dialect/pcre/lexer.lx

```diff
@@ -275,12 +278,28 @@
 	/[0-9]+/ -> $count;
 }
 
+'#'  -> $maybe_comment;
+
+" "  -> $whitespace;
+"\t" -> $whitespace;
+"\v" -> $whitespace;
+"\f" -> $whitespace;
+"\n" -> $newline;
+"\r" -> $newline;
```

Comment:
for new tokens, we should be marking them as unused in the .sid files for all dialects (by prefixing `!` in the `%terminals%` section).

Reply:
These tokens are never seen by the parser, though.  They're either consumed in the function called by `ADVANCE_LEXER` or they're transformed into `$char`.

Reply:
It doesn't hurt to declare them, though.  I've gone ahead and done it.

Reply:
hm, interesting point. not sure how I feel about it, I suppose I'll see how I feel whenever I look at this in the future